test/quantization/test_quantize_fx.py

```diff
@@ -3960,23 +3957,26 @@ def forward(self, x):
         expected_occurrence = {
             # input and weight of first and second linear, output of first and second linear
             ns.call_module(torch.quantization.MinMaxObserver): 6,
+            ns.call_module(torch.quantization.PlaceholderObserver): 1
         }
         self.checkGraphModuleNodes(
             m,
             expected_node_occurrence=expected_occurrence
         )
-        # make sure it runs
         m = convert_fx(m)
         expected_occurrence = {
             ns.call_function(torch.quantize_per_tensor): 2,
-            ns.call_method("dequantize"): 2,
+            # dequantize after first linear, before reshape and before output
+            ns.call_method("dequantize"): 3,
```

Comment:
hmm, sorry, so is the graph actually changing (since these tests are updated with different invocation counts)?

Reply:
Oh, right, I updated the summary to talk about these two test cases, these are incorrect before actually, this PR fixes them.